### PR TITLE
Bug 1723848 Add required registry oreg_auth variables

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1723848

### CODE REVIEW

**Comment by bmcelvee** [comment]
@dongboyan77 PTAL, thanks!

**Comment by bmcelvee** [comment]
@openshift/team-documentation Please peer review, thanks!

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -975,9 +975,21 @@ xref:../install_config/redeploying_certificates.adoc#install-config-redeploying-
...
 == Configuring a Registry Location
 
-If you use an image registry other than the default at
-`registry.redhat.io`, specify the registry in the
-*_/etc/ansible/hosts_* file.
+If you use the default registry at `registry.redhat.io`, you must set the following
+variables:
+
+----
+oreg_url=registry.redhat.io/openshift3/ose-${component}:${version}
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: Please make the variable format consistent in this code block. All the parameter values should be enclosed in [angle brackets](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#command-syntax).

File: install/configuring_inventory_file.adoc [hunk 2]
```diff
@@ -975,9 +975,21 @@ xref:../install_config/redeploying_certificates.adoc#install-config-redeploying-
...
+oreg_auth_user="<user>"
+oreg_auth_password="<password>"
+----
+
+For more information on setting up the registry access token, see
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: s/on/about

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks, then LGTM

**Pull request merged**